saleor/graphql/payment/mutations/payment/checkout_payment_create.py

```diff
@@ -175,9 +175,10 @@ def validate_return_url(cls, input_data):
                 {"redirect_url": e}, code=PaymentErrorCode.INVALID.value
             ) from e
 
+    # TODO This should be unified with metadata_manager and MetadataItemCollection
```

Comment:
Should it be fixed in this PR?

Reply:
No

Reply:
If we have a separate task to fix maybe it would be a good idea also to write down his ID next to this comment?